docs/tasks/stateful-sets/debugging-a-statefulset.md

```diff
@@ -0,0 +1,78 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to debug a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
```

Comment:
Maybe word this as: You should already have a StatefulSet running that you want to investigate. You could link to the StatefulSet concept page too.